e2e/provision/playbooks/cluster.yml

```diff
@@ -96,6 +96,16 @@
     - bootstrap
     - install
   tasks:
+
+    # Git cloning is deliberately done directly on the localhost instead of a pod running in a management cluster.
+    # This way we only need to assume that the localhost can reach the Internet (i.e. http proxy vars are setup correctly),
```

Comment:
```suggestion
    # This way we only need to assume that the host machine can reach the Internet (i.e. http proxy vars are setup correctly),
```

Reply:
this comment was removed along with all changes to cluster.yaml, when I migrated all mirroring into the install role (inspired by your previous commit).